docs/source/conf.py

```diff
@@ -139,7 +139,9 @@ def _get_commit_sha() -> str:
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'examples/imagenet/README.md']
+exclude_patterns = [
+    '_build', 'Thumbs.db', '.DS_Store', 'examples/imagenet/README.md', 'examples/segmentation/README.md'
```

Comment:
@eracah is it okay for me to ignore the README.md here? the bash section in the readme leads to issues with doctests which I wasn't sure how to resolve